src/modules/postReserve.js

```diff
@@ -0,0 +1,25 @@
+import * as resInterface from './reserveInterface.js';
+
+// const projectId = 'HrIKPRrYjrxS00NlIVCD';
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/HrIKPRrYjrxS00NlIVCD/reservations/';
```

Comment:
- [ ] [REQUIRED] When I try to make a reservation, I get this error
<img width="1255" alt="Screenshot 2023-06-10 at 22 42 40" src="https://github.com/RicardoEgas/Kanban-board/assets/58692590/d6aaebdc-6c1a-47bd-a9b4-f3e3f13c765d">

- Hint to resolve
1. Disable any browser extensions that may interfere with the request.
2. Verify that the URL is correct and accessible. You can try opening it directly in your browser and see if it returns the expected response.
3. Test the request in a different browser or an incognito/private browsing window to see if the issue persists.
 - [ ] [REQUIRED] In order to ensure that the UI for the reservation looks like the provided design template, kindly add a close (X) button at the right top of the page so a user will be able to close the popup after making a reservation.
 
 ### Your Design
 
<img width="1423" alt="Screenshot 2023-06-10 at 22 46 31" src="https://github.com/RicardoEgas/Kanban-board/assets/58692590/2aa07e98-5403-437c-b518-98a9ea459948">
### Template Design
<img width="981" alt="Screenshot 2023-06-10 at 22 49 24" src="https://github.com/RicardoEgas/Kanban-board/assets/58692590/806bb976-0520-421b-b564-549663126198">

 
<img width="1423" alt="Screenshot 2023-06-10 at 22 46 31" src="https://github.com/RicardoEgas/Kanban-board/assets/58692590/7b1b6cf0-2aa4-4bfc-8cab-4fd21b372d7b">